src/0_config.js

```diff
@@ -15,3 +15,10 @@ var CORDOVA_BUILD = false; // jshint unused:false
 
 /** @define {boolean} */
 var TITANIUM_BUILD = false; // jshint unused:false
+
+var isApp = (document.URL.indexOf('http://') === -1) && (document.URL.indexOf('https://') === -1);
```

Comment:
looks great. 

1) i might do an alert, although
2) i would rather check window.cordova instead - this would prevent anyone from running it outside the context of http